freqtrade/optimize/hyperopt.py

```diff
@@ -99,6 +106,26 @@
 }
 
 
+def save_trials(trials, trials_path=TRIALS_FILE):
+    "Save hyperopt trials to file"
```

Comment:
A docstring should be enquoted with triple quotes, e.g.:
```
def kos_root():
    """Return the pathname of the KOS root directory."""
    global _kos_root
    if _kos_root: return _kos_root
    ...
```
Same for some other functions in this PR